resolver-dns-native-windows/src/main/c/src/netty_window_dns_native.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* DO NOT EDIT THIS FILE - it is machine generated */
+#include <jni.h>
+/* Header for class io_netty_resolver_dns_windows_NativeWrapper */
+
+#ifndef _Included_io_netty_resolver_dns_windows_NativeWrapper
+#define _Included_io_netty_resolver_dns_windows_NativeWrapper
+#ifdef __cplusplus
```

Comment:
as we dont use c++ anymore I think this cane be removed.

Reply:
Removed.